src/Console/Application.php

```diff
@@ -78,8 +78,7 @@ public function doRun(InputInterface $input, OutputInterface $output)
     {
         $stdErr = $output instanceof ConsoleOutputInterface
             ? $output->getErrorOutput()
-            : ($input->hasParameterOption('--format', true) && 'txt' !== $input->getParameterOption('--format', null, true) ? null : $output)
```

Comment:
IMO this change should be reverted, i.e. the semicolon should remain on a newline. Same with similar changes in other files.

Reply:
Why? This is behavior of `no_multi_line` strategy and this is not chaned calls.

Reply:
Ah right, I was actually expressing a feature request, out of scope here :)